### PR TITLE
Implemented a working production UI dockerfile for Bassa

### PR DESCRIPTION
**Pull request by kmehant**
## Description
Made changes to production Dockerfile for Bassa UI. I worked on getting the built folder **dist** in the initial docker build stage and copied that to nginx container and finally had used nginx server to host the Bassa UI static site pages at `localhost:80`

**Image size: ~22MB**
## Related Issue
#796 

## Motivation and Context

We need a working production level Dockerfile for Bassa UI.

## How Has This Been Tested?

This has been tested locally using docker and docker-compose

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by kmehant** [comment]
@agentmilindu @vivonk Please review!

**Comment by kmehant** [comment]
Fixes #745

File: ui/Dockerfile [hunk 1]
```diff
@@ -1,27 +1,35 @@
-FROM node:9 as builder
```
Comment (JaDogg) [inline ui/Dockerfile]: Why was the node 9 changed to 8?

Comment (kmehant) [reply ui/Dockerfile]: Hi @JaDogg please look in to [this](#375) we have issues with nodesass and higher versions of node.

Comment (VibhorCodecianGupta) [reply ui/Dockerfile]: @kmehant node-sass release notes state that it works with node version between 6 & 7. Is version 8 working fine?

Comment (kmehant) [reply ui/Dockerfile]: @VibhorCodecianGupta yes, version 8 is working fine.

**Pull request merged**